src/sql/gazetteer_lol_extract_view.sql

```diff
@@ -9,7 +9,9 @@
 --
 -- ################################################################################
 
-﻿CREATE OR REPLACE VIEW gazetteer.name_export_for_lol AS
+set search_path= gazetteer, public;
```

Comment:
There's an extra space here and probably should capitlize SET as a convention? 
e.g. `SET search_path=gazetteer, public;`

Reply:
for some reason the convention on this project is lower case for search paths

for example
* https://github.com/linz/gazetteer/blob/85893fcef0fd90f78e325b7943a29f2f058fbed9/src/sql/gazetteer_export_func.sql#L14
* https://github.com/linz/gazetteer/blob/85893fcef0fd90f78e325b7943a29f2f058fbed9/src/sql/gazetteer_export_schema.sql#L13

Reply:
though I just found some that are uppercase :confused:

Reply:
Since there appears to be no convention I updated this to `SET` (from `set`). The extra space has also been removed (so)